templates/projects/hello-world/data.js

```diff
@@ -22,10 +22,39 @@ template.server = {
 
   modelConfigs: [
     {
+      name: 'User',
+      dataSource: 'db',
+    },
+    {
+      name: 'AccessToken',
+      dataSource: 'db',
+      public: false,
+    },
+    {
+      name: 'ACL',
+      dataSource: 'db',
+      public: false,
+    },
+    {
+      name: 'RoleMapping',
+      dataSource: 'db',
+      public: false,
+    },
+    {
+      name: 'Role',
+      dataSource: 'db',
+      public: false,
+    },
+    {
       name: 'Message',
       dataSource: null,
     },
   ],
 
-  datasources: [],
+  datasources: [ 
+    {
+      name: 'db',
+      connector: 'memory',
+    },
+  ],
```

Comment:
I believe @ritch explicitly wanted an example where there are no datasources. @ritch are you ok with modifying this "hello world" template to include in-memory datasource and the whole ACL setup?

If we go down this route, then I think we should also extend the hello-world endpoint to require some authentication, e.g. allow only authorized users to invoke the method.

Thoughts?

Last but not least, I'd like to see unit-tests demonstrating ACLs in hello-world app.